### PR TITLE
Override default iOS styling for inputs (ie, fix search bar) + fix document links on mobile

### PR DESCRIPTION
**Pull request by pcraig3**
A few tiny changes after looking through the frontend toolkit components on an iPhone.

[Trello card with the search bar bug](https://trello.com/c/svwLalaz/153-form-inputs-on-g-cloud-release-when-looking-on-iphone-don-t-appear-to-be-as-intended).

### 1. Square corners on the G-Cloud search bar

| Before | After |
|--------|-------|
| screen shot 2017-05-24 at 16 19 00  | screen shot 2017-05-24 at 16 17 00  |

### 2. Square corners on text inputs

This one is much less noticeable, but there was a slight gap between the outline and the border when focusing on a text input.

| Before | After |
|--------|-------|
| screen shot 2017-05-25 at 10 02 49 | screen shot 2017-05-25 at 10 03 08 |

### 3. More space between document links on mobile

This isn't an iOS problem specifically -- it happens on all small screens.

| Before | After |
|--------|-------|
| screen shot 2017-05-25 at 09 43 24 | screen shot 2017-05-25 at 09 43 49 |

### CODE REVIEW

**Comment by pcraig3** [comment]
[Form inputs on g-cloud release when looking on iPhone don't appear to be as intended](https://trello.com/c/svwLalaz/153-form-inputs-on-g-cloud-release-when-looking-on-iphone-don-t-appear-to-be-as-intended)

File: toolkit/scss/_document.scss [hunk 1]
```diff
@@ -22,7 +22,11 @@
   @include core-19;
   @include box-sizing(border-box);
   @include inline-block;
-  @extend %question-meta-spacing;
+
+  /* Same as values as `question-meta-spacing` but preserved on mobile */
```
Comment (TheDoubleK) [inline toolkit/scss/_document.scss]: Why doesn't the `@extend` work for mobile?  Seems weird.

**Comment by TheDoubleK** [comment]
Nice :+1:

File: toolkit/scss/_document.scss (hunk 1 above)
Comment (pcraig3) [inline toolkit/scss/_document.scss]: The extend brings in some rules that only work on large sizes.  We want them all the time.

**Pull request merged**